src/config.sh

```diff
@@ -223,6 +237,10 @@ function parse_config_options()
         options_values['SCOPE']='local'
         shift
         ;;
+      --show | -s)
+        options_values['IS_SHOW_CONFIGURATIONS']=true
+        shift
+        ;;
```

Comment:
You can probably use something like this:
```bash
options_values['IS_SHOW_CONFIGURATIONS']=1
shift

if [[ -n "$2" ]]; then
  options_values['IS_SHOW_CONFIGURATIONS']="$2" # You can store the new parameter here
  shift
fi
```

P.s.: This is just a draft, I did not test it.

Reply:
I see. However, my implementation accepts more than one config target (`kw config -s vm notification` shows the options for both targets). By following your suggestion I don't see how to mantain this behaviour. Should I remove it and just deal with all or one target?